fix: Geekble Nano board setup update

SooDragon opened the pull request:
Geekble Nano board setup update

## Description of Change
Geekble Nano board setup update

## Tests scenarios
Tested under Arduino IDE 2.3.4 with Geekble nano 2.0 Dev Board

## Related links

SooDragon commented:
In my environment, the old name Geekble_Nano_ESP32S3 resulted Errors, but new name Geekble_nano_ESP32S3 is fine.
I tested it agian but in Arduino IDE 2.3.4, it's same.
It's weird but Geekble_nano_ESP32S3 works, Geekble_Nano_ESP32S3 troubles

may I keep the new name?


and also, reason for commented line was for Heavy-Users.
I removed most of Options in Tools menu for beginners.
but for Heavy Users, I thought they need some of the options I removed for beginners.
So, I left it Commented instead of erasing it, might helpful for Heavy users who want to edit and configure by them selves.

may I also keep the commented lines please?

Thank you! @P-R-O-C-H-Y

P-R-O-C-H-Y commented:
Can you add some log with the other naming (original) which is having troubles?

P-R-O-C-H-Y commented on boards.txt:
```diff
@@ -40846,193 +40846,192 @@ Geekble_ESP32C3.menu.EraseFlash.all.upload.erase_cmd=-e
+Geekble_nano_ESP32S3.build.target=esp32s3
+Geekble_nano_ESP32S3.build.mcu=esp32s3
+Geekble_nano_ESP32S3.build.core=esp32
+Geekble_nano_ESP32S3.build.variant=Geekble_nano_ESP32S3
```
The variant folder is named: `Geekble_Nano_ESP32S3`
Changing this line breaks the board to be working correctly.

SooDragon replied:
Sorry for questioning... @P-R-O-C-H-Y 

1. I Checked Variant folder named: `Geekble_Nano_ESP32S3`
2. `Geekble_nano_ESP32S3.build.variant=Geekble_Nano_ESP32S3`

but Arduino IDE's Tools menu breaks... may I ask what had I done wrong?
I also really don't want to change the board name and tried to understand what you tought me, but I couldn't figured out...
![SCR-20250321-chdu](https://github.com/user-attachments/assets/24109167-8016-45d6-b13a-b2d890d23384)
![SCR-20250321-chzy-3](https://github.com/user-attachments/assets/0886cfa2-4240-449b-a3c0-7e927030cefe)

P-R-O-C-H-Y commented:
About the commented lines, please remove those to keep it clear. None of the boards have huge commented parts. 
You can add to this PR a link to your repo with the "custom" board definition having all those options. Or keep atlas's some of the most used ones :)

SooDragon commented:
@P-R-O-C-H-Y I erased the comment lines and and added logs. Thank you!

Jason2866 commented:
Where is the log file?

SooDragon commented:
> Where is the log file?

On the description :)

P-R-O-C-H-Y commented:
> > Where is the log file?
> 
> On the description :)

I dont see any error log there, which is reasoning the name change.

SooDragon commented:
> > > Where is the log file?
> > 
> > 
> > On the description :)
> 
> I dont see any error log there, which is reasoning the name change.

fix log : Changed Board name : Geekble_Nano_ESP32S3 to Geekble_nano_ESP32S3 to resolve Tools menu error

was What I added, sorry for repeated requests, is there any suggestions or sample for adding error log?
I thought what I commented was a log for the error

Jason2866 commented:
For the boards name change there needs to be a reason. Post the log showing it fails using the not changed original board name.

SooDragon commented:
> For the boards name change there needs to be a reason. Post the log showing it fails using the not changed original board name.

I Updated the log!
Could you please check it out?

P-R-O-C-H-Y commented:
I have no issues with compiling the Geekble nano S3 as it is now without your changes.

SooDragon commented:
The changes are focused on Helping Arduino Beginners, So with my changes, They will see less Tools options might confuse them :)

SooDragon commented:
Yes It's applied could you check it out?
1. commented lines are removed
2. name is changed to resolve error described above
3. removed unnecessary Tools menu so not to confuse Arduino Beginners
Those 3 had been applied :)

P-R-O-C-H-Y commented:
> Yes It's applied could you check it out?
> 
> 1. commented lines are removed
> 2. name is changed to resolve error described above
> 3. removed unnecessary Tools menu so not to confuse Arduino Beginners
>    Those 3 had been applied :)

@SooDragon You are still renaming, which is not necessary as I messaged. I did test the board with the "old" name and there is no issue. So please revert the naming back and do only the 1. and 3. point for your comment. Also please check again if the variant folder name matches the variant name in boards.txt

SooDragon commented:
Thank you for your patience and support.
I changed following items according to your comments.

1. removed re-naming and back to old name `Geekble_Nano_ESP32S3`
2. checked variants folder name, and it was "Geekble_Nano_ESP32S3" so it's applied`Geekble_Nano_ESP32S3.build.variant=Geekble_Nano_ESP32S3
`

Thank you! @P-R-O-C-H-Y
